cpp/src/io/csv/csv_reader.cu

```diff
@@ -1075,23 +1106,47 @@ __global__ void convertCsvToGdf(
 					{
 						int8_t *gdf_out = (int8_t *)gdf_data[actual_col];
 						gdf_out[rec_id] = convertStrtoInt<int8_t>(raw_csv, start, tempPos, opts.thousands);
+
```

Comment:
This `switch` should be updated to use the `type_dispatcher`: https://github.com/rapidsai/cudf/blob/master/cpp/src/utilities/type_dispatcher.hpp#L92

Reply:
Updating to use the `type_dispatcher` is probably outside the scope of this PR, just wanted to let you know that the `type_dispatcher` exists and alleviates the need for these long ugly switches all over.